src/model/quotes.js

```diff
@@ -279,20 +274,31 @@ class QuotesModel {
       let res
       try {
         res = await fetch(fullUrl, opts)
-      } catch (e) {
-        throw new Errors.FSPIOPError('Network error', `Network error forwarding quote request: ${e.stack || util.inspect(e)}`,
-          fspiopSource, Errors.ApiErrorCodes.DESTINATION_COMMUNICATION_ERROR)
+      } catch (err) {
+        throw ErrorHandler.CreateFSPIOPErrorFromErrorCode(1001, `Network error forwarding quote request to ${fspiopDest}`, err, fspiopSource, [
```

Comment:
Please use the enum for errorCode from the Error library

Reply:
Done